src/VisualStudio/Core/Test/Progression/SearchGraphQueryTests.vb

```diff
@@ -252,5 +320,23 @@ End Namespace
                     </DirectedGraph>)
             End Using
         End Sub
+
+        ' IProgressionPrimaryWorkspaceProvider cannot be used with Moq without lots of <InternalsVisibleTo /> tags so a
+        ' simple class was created instead to wrap it's single property.
+        Private Class MockProgressionPrimaryWorkspaceProvider
```

Comment:
Rather than all of this, it still seems easier if we just test the BeginGetGraphData method directly. If you extract out the logic of context -> IEnumerable<GraphQueries> directly to a simple static method, can't we just write a very simple test that asserts that given the graph query in question the resulting collection contains a ContainsGraphQuery. Much smaller and more direct.